src/robusta/core/sinks/kafka/kafka_sink.py

```diff
@@ -1,19 +1,24 @@
 import json
 import logging
-from kafka import KafkaProducer
 
-from .kafka_sink_params import KafkaSinkConfigWrapper
-from ...reporting.blocks import KubernetesDiffBlock, JsonBlock
-from ...reporting.base import Finding, Enrichment
-from ..sink_base import SinkBase
+try:
+    from kafka import KafkaProducer
+except ImportError:
```

Comment:
We also might just think about introducing some check on if we are in cli app or not
And then import based on that

Reply:
I think I would just add comments in the code where you put the import errors, 
`# these are optional packages in the cli and could not be imported`

Reply:
Yeah, but try solves the runtime ImportError. I think it was not a problem because of wildcard imports and python was lazy-loading those and not making errors in cli. But because I removed all widcard imports - this is now the case.

The only other way to solve ImportErrors is to move imports right in the function, that is using `KafkaProducer`. For annotations it is then possible to use `if TYPE_CHECKING:`